supply-chain-security/compromises/compromise-definitions.md

```diff
@@ -0,0 +1,39 @@
+# Types of Supply Chain Compromise
+
+This document serves to provide a consistent set of definitions, recognized by SIG-Security, for the Supply Chain Compromise Catalog.  The types of compromise may be expanded in the future as new or more distinct compromises occur.  When this happens, these definitions will be updated.
```

Comment:
"recognized by SIG-Security" may be redundant here. I think anything in the repo should be governed by terms of use.

Reply:
corrected